backend/app/controllers/spree/admin/payment_methods_controller.rb

```diff
@@ -7,6 +7,10 @@ class PaymentMethodsController < ResourceController
 
       respond_to :html
 
+      def index
```

Comment:
@krtschmr rather than overriding the `index` action you can add custom `collection` method:

```
def collection
   @collection = super.order(position: :asc)
end
```